docs/localized/KR/docs/API-Reference.md

```diff
@@ -0,0 +1,21 @@
+# API Reference
```

Comment:
please remove any non-translated page (i.e. english only).  this will keep the KR translated documents cleaner.